src/Symfony/Component/Notifier/Bridge/Slack/SlackOptions.php

```diff
@@ -21,7 +21,7 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
-final class SlackOptions implements MessageOptionsInterface
+class SlackOptions implements MessageOptionsInterface
```

Comment:
Should we mark this class internal  now?

Reply:
> Classes, properties and methods that bear the tag @internal as well as the classes located in the various *\Tests\ namespaces are an exception to this rule. They are meant for internal use only and should not be accessed by your own code.

No, I don't think so, this class is used in user code, and should be (when posting a message with blocks, or sending reply to a thread)